modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/css/main.scss

```diff
@@ -1,4 +1,17 @@
 .portlet-client-extension-admin {
+	.inline-item-after {
+		margin-left: 0.25rem;
+	}
+
+	.btn-monospaced.btn-xs {
+		height: 1rem;
+		width: 1rem;
+
+		.lexicon-icon.lexicon-icon-plus,
+		.lexicon-icon.lexicon-icon-hr {
+			height: 10px;
+		}
+	}
```

Comment:
I'm not sure if this is the exclusive size for this use case but if it's a general size maybe we should review this in Clay if it's a bug? cc @pat270

Reply:
Yes, I noticed we don't have a `required` property to add on input labels and we might have to add it on Clay

Reply:
Yes, we don't have buttons that small on clay.

Reply:
Yep, this would be good to add to Clay, so we can consistently apply these buttons in other places such as the ddm form field repeatable lists. @drakonux is it okay to add a new button size for the add and remove buttons?
![Screenshot 2024-02-29 at 10 07 24 AM](https://github.com/liferay-platform-experience/liferay-portal/assets/1609196/b1a82fd9-9bd6-43b8-9fca-54422795ec24)

Reply:
XS size (24px) is the only available size. My recommendation is to invest time in the pattern instead of adding a new size (not recommended for accessibility). See the [exploration page](https://www.figma.com/file/AuJCs8t0dINcA5UDpGbej5/LPS-158015-Improve-the-input-component-with-buttons-used-in-mimetypes?type=design&node-id=1%3A2&mode=design&t=iEJqXn2WrBLmhqXh-1) María Martins did for more information

Reply:
Can we keep the + and - buttons at 24px? 16x16 makes it hard to click.

Reply:
In other places that have the same behavior as the attributes component, is used the 16px button.
See an example below:

![Screenshot_20240304_175201](https://github.com/liferay-platform-experience/liferay-portal/assets/47724428/b0ac5f1c-fc53-4132-9df2-72d197f5a8db)

Reply:
Hey @drakonux
Which proposal on [exploration page](https://www.figma.com/file/AuJCs8t0dINcA5UDpGbej5/LPS-158015-Improve-the-input-component-with-buttons-used-in-mimetypes?type=design&node-id=1-2&mode=design&t=NSUZ7BxgmDMFzDUS-0) should I follow about the remove and add buttons?
I liked proposal 2.

cc: @ethib137 @pat270 @matuzalemsteles

Reply:
For this ticket we should leave it as is, but I've created a [feature request](https://liferay.atlassian.net/browse/LPD-19951) to review this pattern so that we can apply a consistent update for this across all repeatable fields.

cc: @drakonux